file_paths.md

```diff
@@ -253,6 +254,7 @@ create a `puppet` user or group.
                 pxp-access.log                # not enabled by default
             spool *                           # directory containing results of pxp-agent modules
             run *
+        tasks-cache *                         # directory containing a cache of files for all downloaded tasks
```

Comment:
Putting the `spool` dir under `var` on Windows looked like a mistake to me. On *nix it'd be under `opt`. However, we could choose to make it consistent even though it's kind of weird.